exporter/kineticaexporter/metadata.yaml

```
@@ -0,0 +1,7 @@
+type: kinetica
+
+status:
+  class: exporter
+  stability:
+    development: [metrics, traces, logs]
+  distributions: [contrib]
```

```suggestion
  distributions: []
```

This is not included in the official Contrib distribution yet

---

Modified as suggested